CONTRIBUTING.md

```diff
@@ -36,3 +36,51 @@ When requesting or submitting new features, first consider whether it might be u
 - Commit Unix line endings.
 - Please use the Pi-hole brand: **Pi-hole** (Take a special look at the capitalized 'P' and a low 'h' with a hyphen)
 - (Optional fun) keep to the theme of Star Trek/black holes/gravity.
+
+## Forking and Cloning from GitHub to GitHub
+
+1. Fork https://github.com/pi-hole/pi-hole/ to a repo under a namespace you control, or have permission to use, (example: [https://github.com/<your_namespace>/pi-hole/]()).  You can do this from the github.com website.
+2. Clone [https://github.com/<your_namespace>/pi-hole/]() with the tool of you choice.
+3. To keep your fork in sync with our repo, add an upstream remote for pi-hole/pi-hole to your repo.
+    ```git remote add upstream https://github.com/pi-hole/pi-hole.git```
```

Comment:
```suggestion
    `git remote add upstream https://github.com/pi-hole/pi-hole.git`
```

Reply:
I changed it to a code block so the command would stand out.